src/util/sandbox.cpp

```diff
@@ -385,66 +401,76 @@ QString Sandbox::migrateOldSettings() {
     // outside the sandbox, unfortunately it is necessary to annoy the user with this popup
     // even if they are installing Mixxx >= 2.3.0 without having installed an old version of Mixxx.
     QString title = QObject::tr("Upgrading old Mixxx settings");
-    QString oldPath = homePath + QLatin1String("/Library/Application Support/Mixxx");
     QMessageBox::information(nullptr,
-            title,
-            QObject::tr(
-                    "Due to Mac sandboxing, Mixxx needs your permission to "
-                    "access your music library "
-                    "and settings from Mixxx versions before 2.3.0. After "
-                    "clicking OK, you will see a file picker. "
-                    "To give Mixxx permission, press the Ok button in the file "
-                    "picker."
-                    "\n\n"
-                    "If you do not want to grant Mixxx access click Cancel on "
-                    "the file picker."));
+        title,
+        QObject::tr(
+            "Existing Mixxx settings directory must be moved to new location."
+            "\n\n"
+            "Due to MacOS sandboxing, Mixxx needs your permission to access your "
```

Comment:
```suggestion
            "Due to macOS sandboxing, Mixxx needs your permission to access your "
```
following Apple's styling